libvirt/defaults.yaml

```diff
@@ -11,7 +11,7 @@ Debian:
     networks_dir: {}
     network_file: {}
   service:
-    name: libvirt-bin
+    name: libvirtd
```

Comment:
this will break wheezy compatibility. you could use something like the following but for the OS code name instead of the OS name.

* https://github.com/bechtoldt/saltstack-mongodb-formula/blob/master/mongodb/defaults.yaml#L22
* https://github.com/bechtoldt/saltstack-mongodb-formula/blob/master/mongodb/server.sls#L5

Nevertheless I'll update the formula to use https://github.com/bechtoldt/salt-modules/blob/master/_modules/formhelper.py. Unfortunately this means users will need to install the formhelper execution module.